capstone/src/main/java/com/google/sps/servlets/scraperServlet.java

```diff
@@ -52,4 +56,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(restaurantNamesJson);
   }
-}
\ No newline at end of file
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Sorry, a little confused here. This file looks a lot like capstone/src/main/java/com/google/sps/servlets/BlackOwnedRestaurantsDataServlet.java. What's the difference between the two?

Reply:
`scraperServlet.java` is for building the small business database while `BlackOwnedRestaurantsDataServlet.java` is for building the black owned business database

Maybe updating the name of the Servlet to reflect this would be more clear?

Reply:
that makes sense, I'll make that change

Reply:
Got it, thanks!